handlers/token_api.py

```diff
@@ -24,6 +24,9 @@ def get_tokens(
     search_after = event.query.get("search_after") or ""
     search_after_list = []
 
+    if sort_by == "uid":
+        sort_by = "id"
```

Comment:
This is to support requests from the Tokens balance screen as well, right? Should we add a comment explaining it?

Reply:
No, actually this was a design decision done on the token list API, I just moved this if outside of the `elastic_search_utils` method

The idea is to allow the user to sort by either `uid` or `id` (they are the same thing)

I removed it from `elastic_search_utils` `build_search_query`to be able to generalize it